src/sentry/models/dashboard.py

```diff
@@ -152,6 +152,7 @@ class Meta:
                 {
                     "title": "Errors by Country",
                     "displayType": "world_map",
+                    "interval": "5m",
```

Comment:
Oh was this not caught because this isn't typed?

Reply:
Probably not. I'm not totally familiar with typing in Python.